packages/patternfly-4/react-core/src/experimental/components/OverflowMenu/OverflowMenuDropdownItem.tsx

```diff
@@ -0,0 +1,26 @@
+import * as React from 'react';
+import { css } from '@patternfly/react-styles';
+import styles from '@patternfly/react-styles/css/components/OverflowMenu/overflow-menu';
+import { DropdownItem } from '../../../components/Dropdown';
+
+export interface OverflowMenuDropdownItemProps extends React.HTMLProps<HTMLDivElement> {
+  /** Any elements that can be rendered in the menu */
+  children?: any;
+  /** Additional classes added to the OverflowMenuDropdown */
+  className?: string;
+  /** Indicates when a dropdown item shows and hides the corresponding list item */
+  shared?: boolean;
```

Comment:
boolean public props should be prefixed, e.g. `isShared`.

Reply:
Updated prop name to isShared